client/components/select-dropdown/style.scss

```diff
@@ -130,6 +138,9 @@ $compact-header-height: 35;
 	border-top: 0;
 	// $blue-wordpress for outer (with focus shadow), $gray for border with header
 	border-radius: 0 0 4px 4px;
+	height: 0;
+	overflow: hidden;
```

Comment:
@retrofox 

I guess this could look strange without a background story.
Initially I wrote this as a simply `display: none;` but I found that, this affected the width of the component, causing sometimes overlap between different elements.
This version is much verbose but has not not-desidered effects.

Please note: I've tested only on OS Sierra with Chrome, Firefox and Safari.